data/scripts/talkactions/gm/broadcast.lua

```diff
@@ -11,7 +11,7 @@ function broadcast.onSay(player, words, param)
 
 	local text = player:getName() .. " broadcasted: " .. param
 	logger.info(text)
-	Webhook.sendMessage("Broadcast", text, WEBHOOK_COLOR_WARNING, announcementChannels["serverAnnouncements"])
+	--Webhook.sendMessage("Broadcast", text, WEBHOOK_COLOR_WARNING, announcementChannels["serverAnnouncements"])
```

Comment:
```suggestion
	Webhook.sendMessage("Broadcast", text, WEBHOOK_COLOR_WARNING, announcementChannels["serverAnnouncements"])
```

Reply:
In a discussion with Edu, we decided to comment on this function because the webhook sendMessage function isn't encoding accentuation characters.

Reply:
What happens? Tibia also has issues showing accents, but I'd still rather be able to broadcast webhooks in english